src/main/scala/com.socrata.tileserver/util/TileEncoder.scala

```diff
@@ -1,5 +1,8 @@
 package com.socrata.tileserver.util
 
+import com.rojoma.json.v3.util.JsonUtil
+import com.socrata.soql.environment.ColumnName
```

Comment:
These should live in the same paragraph as the other `com.rojoma` imports (and also be alphabetized, pretty please.  :))